components/tools/OmeroPy/src/omero/plugins/search.py

```diff
@@ -94,6 +94,9 @@ def search(self, args):
         import omero.all
 
         if args.index:
+            if not self.ctx.get_event_context().isAdmin:
+                raise Exception("Only admin can index object")
```

Comment:
Sorry, I was rushing earlier in my pseudo-code. To match the style of the plugins, this should use `self.ctx.die`

Reply:
ok I will adjust accordingly